pages/verify-eth/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -0,0 +1,272 @@
+import { useCallback, useEffect, useState } from "react";
```

Comment:
It's a good idea to separate things, but please write reusable components! There is so much duplicated code, and it'll be very hard if we want to add a new functionnality on ten networks if each of them is written separately.
Ideally, in the pages, there should only be custom components and hooks, where you provide custom parameters. 
For example, if we want to change a bit the text, or add new buttons etc, we only need to do it in one place